config/site_settings.yml

```diff
@@ -529,9 +529,6 @@ users:
     default: false
     client: true
   log_personal_messages_views: false
-  ignore_user_enabled:
```

Comment:
We need a migration to remove that value form the database (`site_settings` table).